executor/app.py

```
@@ -134,6 +134,17 @@ async def lifespan(app: FastAPI):
     except Exception as e:
         logger.warning(f"Failed to start heartbeat service: {e}")
 
+    # Initialize Claude Code for sandbox mode if AUTH_TOKEN is provided
+    # This allows sandbox containers to download skills at startup
+    try:
+        auth_token = os.getenv("AUTH_TOKEN")
+        task_id = os.getenv("TASK_ID")
+        if auth_token and task_id:
+            logger.info(f"AUTH_TOKEN found, initializing Claude Code for sandbox {task_id}...")
+            _initialize_sandbox_claude(auth_token, task_id)
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)